### PR TITLE
#317 | Disabled widgets styling fixed

### PR DESCRIPTION
**Pull request by szymon-owczarzak**
## Description

See related bug #317 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](https://github.com/wttech/cogboard/blob/master/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] Automated functional tests have been added or modified to cover my changes (if applicable)
- [ ] I have updated the documentation accordingly.

---

I hereby agree to the terms of the Cogboard Contributor License Agreement.

### CODE REVIEW

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -25,8 +25,9 @@ jobs:
         run: |
           export VERSION=$(sed -n '1s/^.*[^0-9]\([0-9]*\.[0-9]*\.[0-9]*\).*$/\1/p' gradle.properties)
           echo "::set-output name=current_version::$VERSION"
-      - name: Upload Release asset
+      - name: Upload Release Asset
         id: upload-release-asset
+        continue-on-error: true
```
Comment (szymon-owczarzak) [inline .github/workflows/release-drafter.yml:30]: Additional fix for release drafter

File: cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/zabbix/ZabbixWidget.kt [hunk 2]
```diff
@@ -61,7 +61,7 @@ class ZabbixWidget(
     }
 
     private fun extractResult(responseBody: JsonObject): Any {
-        val body = responseBody.getString(BODY)
+        val body = responseBody.getString(BODY) ?: "{}"
```
Comment (szymon-owczarzak) [inline cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/zabbix/ZabbixWidget.kt:64]: NPE fix

File: cogboard-webapp/src/components/Widget/index.js [hunk 3]
```diff
@@ -51,12 +51,14 @@ const Widget = ({ id, index }) => {
...
-    ? false
-    : isExpandContent;
+      ? defaultExpandedContent
+      : type === 'TextWidget' && isVertical
+      ? false
+      : isExpandContent;
   const widgetTypeConfig = widgetTypes[type] || widgetTypes['WhiteSpaceWidget'];
-  const widgetStatus = getWidgetStatus(content, widgetTypeConfig);
+  const widgetStatus = disabled
+    ? 'DISABLED'
```
Comment (szymon-owczarzak) [inline cogboard-webapp/src/components/Widget/index.js:60]: when disabled status is set to `DISABLED`

**Review by yahorm — APPROVED** [review]

File: cogboard-webapp/src/components/Widget/styled.js [hunk 4]
```diff
@@ -167,8 +173,8 @@ export const StyledCollapse = styled(
     !isDragging && type !== 'ZabbixWidget'
       ? mapStatusToColor(status, theme)
       : type === 'ZabbixWidget'
-        ? mapStatusToColor('UNKNOWN', theme)
-        : theme.palette.background.paper};
+      ? mapStatusToColor('UNKNOWN', theme)
+      : theme.palette.background.paper};
```
Comment (szymon-owczarzak) [inline cogboard-webapp/src/components/Widget/styled.js:177]: Wrong indetation

**Pull request merged**